mitodl/bootcamp-ecommerce
# Admissions section - #485

#### What are the relevant tickets?
#485 

#### What's this PR do?
- Adds an `AdmissionsSectionPage` to the `BootcampRunPage` children. Displays the "admissions" section on the page.
- Fixes HTML structure of three column section
- Fixes HTMl structure and SCSS of instructor carousel

#### How should this be manually tested?
Add the appropriate information and visit the bootcamp page to verify everything appears as it should.


#### Screenshots (if appropriate)
Wagtail UI
![screencapture-bc-odl-local-8099-cms-pages-13-edit-2020-05-27-17_01_22](https://user-images.githubusercontent.com/6387678/83016657-c06b6600-a03b-11ea-889e-848ed64babb8.png)


Desktop
<img width="1667" alt="Screenshot 2020-05-27 at 4 12 44 PM" src="https://user-images.githubusercontent.com/6387678/83016518-826e4200-a03b-11ea-96df-53a6aee5acd2.png">


Tablet
<img width="515" alt="Screenshot 2020-05-27 at 4 14 01 PM" src="https://user-images.githubusercontent.com/6387678/83016539-8dc16d80-a03b-11ea-901d-d8114da886ed.png">

Mobile
![screencapture-bc-odl-local-8099-bootcamps-bootcamp-23-2020-05-27-16_57_28](https://user-images.githubusercontent.com/6387678/83016554-95811200-a03b-11ea-8a1d-f88e76047674.png)
